storage/backend-test-suite/src/prelude.rs

```diff
@@ -46,7 +46,8 @@ pub fn using_proptest<B: Backend, F: BackendFn<B>, S: proptest::prelude::Strateg
 ) {
     let config = {
         let mut config = proptest::prelude::ProptestConfig::with_source_file(source_file);
-        config.cases = 64;
+        // Decrease the number of test cases. By default, this is 256 / 8 = 64.
```

Comment:
Why? Can you please provide some context?

Reply:
By default, this is the same as was before. I just wanted to have the ability to adjust the amount using the `PROPTEST_CASES` env variable.